homeassistant/components/arcam_fmj/.translations/en.json

```diff
@@ -1,5 +1,28 @@
 {
     "config": {
-        "title": "Arcam FMJ"
+        "title": "Arcam FMJ",
+        "step": {
+            "user": {
+                "title": "Connect to Arcam FMJ",
+                "description": "Please enter the host name or IP address of device.",
+                "data": {
+                    "uuid": "Unique identifier",
+                    "host": "Host",
+                    "port": "Port"
+                }
+            }
+        },
+        "error": {
+            "connection_failure": "Unable to connect to the specified host."
+        },
+        "abort": {
+            "already_setup": "Device was already setup.",
+            "updated_instance": "Updated instance with new host address."
+        }
+    },
+    "device_automation": {
+        "trigger_type": {
+            "turn_on": "{entity_name} was requested to turn on"
```

Comment:
Can we split the device automation part out of this PR? 

I also think, although very clever, not resulting in the best user experience. Now they need to realize they have to go to the automations editor to set specific actions.

Reply:
I'll split out the config flow stuff once i have SSDP working nicely.

Do you you have some other thoughts on how to handle this use case thou? Device automations isn't such a bad fit really. Apart from the entity not knowing if there is a connected automation.

Reply:
I don't have right now. We could discuss it in an architecture issue, but there is already so much on the backlog :(